### PR TITLE
XSIAM Cryptomining

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
A response playbook for XCloud cryptomining alert designed for XSIAM.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.6.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+  type: unknown
+- contextPath: IdentityIQ
+  description: The IdentityIQ object.
+  type: unknown
+- contextPath: ActiveDirectory.Users
+  description: The ActiveDirectory Users object.
+  type: unknown
+tests:
+- No tests (auto formatted)
+marketplaces: ["marketplacev2"]
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml:524]: The pack already only for XSIAM, you can remove it from here

File: Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+      - "37"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsGeneral
+          left:
+            value:
+              complex:
+                root: Core.OriginalAlert.event
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml:217]: what about google?

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml [hunk 3]
```diff
@@ -0,0 +1,786 @@
...
+  "27":
+    id: "27"
+    taskid: 0e878c77-92fd-4ca9-81f9-56d977cb10d6
+    type: regular
+    task:
+      id: 0e878c77-92fd-4ca9-81f9-56d977cb10d6
+      version: -1
+      name: Set the incident severity
+      description: Optionally increases the incident severity to the new value if it is greater than the existing severity.
+      scriptName: IncreaseIncidentSeverity
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:77]: Have you checked how this automation runs on XSIAM? will it raise the alert or the incident? (Cause I dont see any input for the incident)

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml [hunk 4]
```diff
@@ -0,0 +1,786 @@
...
+    }
+  }
+inputs:
+- key: incident_id
+  value: {}
+  required: false
+  description: The incident ID.
+  playbookInputQuery:
+- key: alert_id
+  value: {}
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:754]: shouldn't we already provide the field for it?

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml [hunk 5]
```diff
@@ -0,0 +1,786 @@
...
+      "dimensions": {
+        "height": 2535,
+        "width": 970,
+        "x": 720,
+        "y": -200
+      }
+    }
+  }
+inputs:
+- key: incident_id
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:748]: Why do you need it? and if we do, lets add a default field (as far as I know, the incident ID of the alert doesnt exist in the alert context.

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml [hunk 6]
```diff
@@ -0,0 +1,786 @@
...
+  required: false
+  description: The alert ID.
+  playbookInputQuery:
+- key: SOCEmailAddress
+  value:
+    simple:
+  required: false
+  description: The SOC email address to use for the alert status notification.
+  playbookInputQuery:
+- key: requireAnalystReview
```
Comment (altmannyarden) [inline Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:764]: isnt it the same as ShouldCloseAutomatically?

File: Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml (hunk 1 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml:524]: done

File: Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml (hunk 2 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-XCloud_Alert_Enrichment.yml:217]: Done

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml (hunk 3 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:77]: Updates the alert's severity. Of course, if the updated severity has been raised (M->H) for example, the incident's severity will be updated as well.

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml (hunk 4 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:754]: No, we provide it in the Alerts Handling playbook.

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml (hunk 5 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:748]: Correct, Removed.

File: Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml (hunk 6 above)
Comment (melamedbn) [reply Packs/Core/Playbooks/playbook-XCloud_Cryptomining.yml:764]: ShouldCloseAutomatically is an input used in the Handle FP playbook and requireAnalystReview is used in the TP flow.

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_Response_-_Generic.yml [hunk 7]
```diff
@@ -422,36 +410,40 @@ tasks:
...
-        simple: "False"
+        complex:
+          root: inputs.autoResourceRemediation
       autoUserRemediation:
-        simple: "False"
+        complex:
+          root: inputs.GCP-accessKeyRemediationType
       resourceName:
         complex:
           root: alertJson._all_events
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Cloud_Response_-_Generic.yml:433]: that one stayed with `alertJson._all_events `context path

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Cloud_Cryptomining.yml [hunk 8]
```diff
@@ -892,6 +850,140 @@ tasks:
...
+      playbookName: Cloud Response - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "40"
+    scriptarguments:
+      AWS-accessKeyRemediationType:
+        simple: Disable
```
Comment (altmannyarden) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Cloud_Cryptomining.yml:881]: Should be taken from the new inputs created, no?

File: Packs/CommonPlaybooks/Playbooks/playbook-Cloud_Response_-_Generic.yml (hunk 7 above)
Comment (melamedbn) [reply Packs/CommonPlaybooks/Playbooks/playbook-Cloud_Response_-_Generic.yml]: fixed

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_27.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
 
 #### Playbooks
-##### Cloud Response - Generic
-- Added an input named 'cloudProvider' for the playbook's conditional task deciding which CSP is involved.
\ No newline at end of file
+##### Search For Hash In Sandbox - Generic
+- Fixed an issue where the playbook failed on the `Get Wildfire PDF report` task due to timeout. Timeout was extended from 5 to 15 minutes.
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_27.md:4]: Dont thinks its related to your PR

**Review by altmannyarden — APPROVED** [review]

File: Packs/CortexXDR/ReleaseNotes/4_10_4.md [hunk 10]
```diff
@@ -1,3 +1,6 @@
+
 #### Playbooks
-##### Cortex XDR - Cloud Cryptomining
-- Added multiple inputs to the Cloud Response - Generic sub-playbook.
\ No newline at end of file
+##### Cortex XDR - Retrieve File by sha256
+Fixed an issue where the checked process path for "actor" was pointing on "action" instead.
```
Comment (altmannyarden) [inline Packs/CortexXDR/ReleaseNotes/4_10_4.md:4]: you didnt touched those playbooks

**Pull request merged**